helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -405,7 +407,68 @@ public void testEnableDisableMaintenanceModeWithCustomFields() {
         accessor.getBaseDataAccessor().exists(accessor.keyBuilder().maintenance().getPath(), 0));
   }
 
-  @Test()
+  @Test(dependsOnMethods = "testEnableDisableMaintenanceModeWithCustomFields")
+  public void testGetStateModelDef() throws IOException {
+
+    System.out.println("Start test :" + TestHelper.getTestMethodName());
+    String cluster = "TestCluster_1";
+    String stateModelDefs = get("clusters/" + cluster + "/statemodeldefs", null, Response.Status.OK.getStatusCode(), true);
+    Map<String, Object> defMap = OBJECT_MAPPER.readValue(stateModelDefs, new TypeReference<HashMap<String, Object>>(){});
+
+    Assert.assertTrue(defMap.size()==2);
+    Assert.assertTrue(defMap.get("stateModelDefinitions") instanceof List);
+    List<String> stateModelNames = (List<String>)defMap.get("stateModelDefinitions");
+    Assert.assertEquals(stateModelNames.size(), 6);
+
+    String oneModel = stateModelNames.get(1);
+    String twoModel = stateModelNames.get(2);
+
+    String oneModelUri = "clusters/" + cluster + "/statemodeldefs/" + oneModel;
+    String oneResult = get(oneModelUri, null, Response.Status.OK.getStatusCode(), true);
+    ZNRecord oneRecord = OBJECT_MAPPER.readValue(oneResult, ZNRecord.class);
+
+    String twoModelUri = "clusters/" + cluster + "/statemodeldefs/" + twoModel;
+    String twoResult = get("clusters/" + cluster + "/statemodeldefs/" + twoModel, null, Response.Status.OK.getStatusCode(), true);
```

Comment:
It seems towModeUri is what you need to replace this string concatenation?

Reply:
fixed.